encrypted_merlin.md

```diff
@@ -0,0 +1,94 @@
+# Encrypted Merlin
+
+It is possible to configure merlin to encrypt the packets sent over the merlin port (default 15551).
+This can be useful if different OP5 Monitor nodes need to communicate over the internet, for example if you have a remote poller.
+For nodes which are placed within the same secure local network, encryption is often unnecessary, and introduces an unnecessary performance penalty.
+
+Merlin uses public key encryption, which requires setup of both a private and public key for each node.
+
+The encryption used is:
+
+Key agreement protocol: ECDH
+Encryption: XSalsa20
+Authentication: Poly1305 MAC
+
+# Setup
+
+## Generate and copy keys
+To setup encryption, you must first generate a key pair on every node involved in the encrypted communication.
+
+This is done using the command `mon merlinkey generate`. This will generate two files:
+key.priv and key.pub at /opt/monitor/op5/merlin by default, or you can use --path=/preferred/path/here
+
+When the keys have been created, the public key (key.pub) needs to be copied to the relevant servers.
+
+This can be done with `scp`, for example, in a situation where we have one master and one poller:
+
+On master:
+scp /opt/monitor/op5/merlin/key.pub poller_ip:/opt/monitor/op5/merlin/master.pub
+
+On poller:
+scp /opt/monitor/op5/merlin/key.pub master_ip:/opt/monitor/op5/merlin/poller.pub
+
+**Important:** The private key (key.priv) should never be shared.
+
+## Change merlin configuration
+
+After the key pairs have been generated and the public keys have been copied we need to configure merlin to use them.
+
+To do so, we must change the merlin configuration file at /opt/monitor/op5/merlin/merlin.conf.
+
+First we must add the "top-level" setting `ipc_privatekey`. I can be added directly after the `ipc_socket` line.
+
```

Comment:
I think we should remove this new line so that this section and the previous section is combined in to one. We then get to sections like

"First we..."

"Secondly we..."